quickjs.c

```diff
@@ -19240,19 +18744,11 @@ static __exception int next_token(JSParseState *s)
             const uint8_t *p1;
             int flags, radix;
             flags = ATOD_ACCEPT_BIN_OCT | ATOD_ACCEPT_LEGACY_OCTAL |
-                ATOD_ACCEPT_UNDERSCORES;
-#ifdef CONFIG_BIGNUM
-            flags |= ATOD_ACCEPT_SUFFIX;
-#endif
+                ATOD_ACCEPT_UNDERSCORES | ATOD_ACCEPT_SUFFIX;
             radix = 0;
```

Comment:
Nit: I think radix is only used on the following function call.

Reply:
Removed!